### ISSUE TITLE
ProcessSendgridWebhookJob error on webhooks not related to the campaign manager

### ISSUE THREAD

**Issue by ryanhungate** [spatie/laravel-mailcoach-sendgrid-feedback#7]
Hey guys - i just noticed a bug when you're not using the campaign builder. Error happens on webhooks that are unrelated to the app itself ( like any other message coming through SendGrid). 

This method on the `ProcessSendgridWebhookJob` class is throwing an error while finding `Send::findByUuid($sendUuid);` with a null value. This code tweak seems to fix the problem and I didn't want to go through the pull requests for something so small.

```
protected function getSend(array $rawEvent): ?Send
{
    $sendUuid = Arr::get($rawEvent, 'send_uuid');

    return empty($sendUuid) ? null : Send::findByUuid($sendUuid);
}
```

**Comment by freekmurze** [comment]
Feel free to submit a PR that fixes this, along with a test

**Comment by rubenvanassche** [comment]
Fixed with 2.2.3
